tests/guardrails/max_temp_basal_rate.tests.js

```diff
@@ -0,0 +1,36 @@
+
+const { limits } = require('../../src/index');
+
+var maxTempBasalRateTests = (test) => {
+    var screen;
+    it('open', async () => {
+        screen = await test.settingsScreen.OpenDeliveryLimitsScreen();
+    });
+    it('set below max limit', async () => {
+        await screen.Apply({
+            maxBolus: '1.0',
+            maxBasalRate: limits.basalDelivery.max.limit - limits.basalDelivery.step
+        });
+        //TODO assert NO warning
+    });
+
+    it('set at max limit', async () => {
+        await screen.Apply({
+            maxBolus: '1.0',
+            maxBasalRate: limits.basalDelivery.max.limit
+        });
+        //TODO assert on warning
+    });
+    it('cannot above max limit', async () => {
```

Comment:
[nit] Put this test on top so we are consistent with the rest of the test going from maximum to minimum?